src/nunavut/jinja/__init__.py

```diff
@@ -792,7 +792,7 @@ def __init__(self, namespace: nunavut.Namespace, **kwargs: typing.Any):
     # +-----------------------------------------------------------------------+
     # | AbstractGenerator
     # +-----------------------------------------------------------------------+
-    def get_templates(self) -> typing.Iterable[pathlib.Path]:
+    def get_templates(self) -> typing.Iterable[tuple[pathlib.Path, pathlib.Path]]:
```

Comment:
Please add `from future import __annotations__` to avoid breakage with old Pythons. https://www.python.org/dev/peps/pep-0585/